lambify/bin/_build

```diff
@@ -29,11 +23,3 @@ echo "== Environments & Configuration =="
 
 echo "== Asset Hosts & Precompiling =="
 NODE_ENV='production' ./bin/rails assets:precompile
-
-echo "== Cleanup Unused Files & Directories =="
-rm -rf \
-  log \
-  node_modules \
-  test \
-  tmp \
-  vendor/bundle/ruby/2.7.0/cache
```

Comment:
This is where the cleanup was moved to the `Dockerfile` so that when the deploy workflow job finished, it would have a good directory to cache.